lte/gateway/c/core/oai/include/sgw_context_manager.hpp

```diff
@@ -23,15 +23,18 @@
  */
 #pragma once
 
+#include <stdint.h>
+
+#include "lte/gateway/c/core/oai/include/spgw_state.hpp"
```

Comment:
This should be after the `extern "C"` block as per the [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)